HW03/aleksei_zhuchkov/spec/hw_03_t_04_spec.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+require_relative '../aleksei_zhuchkov_hw_03_t_04'
+
+RSpec.describe 'Homework3' do
+  let(:obj_hw3) { Homework3.new }
+
+  context 'when testing basic things' do
+    it 'return value is Hash type' do
+      expect(obj_hw3.task4('')).to be_an_instance_of(Hash)
+    end
+
+    it 'return value 0 for numbers and letters when get empty string' do
+      expect(obj_hw3.task4('')).to eq({ digits: 0, letters: 0 })
+    end
+  end
+
+  context 'when testing different kind of requests' do
```

Comment:
you can split it into 2 contexts

Reply:
I have done it
https://github.com/labs-ruby/homeworks-2022/pull/122/commits/90ca4c1dba728c82c10cfd2744bdad68034c55e3